File: week-02/day-01/draw_chess_table.py
# Crate a program that draws a chess table like this
#
# % % % % 
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % % 
#  % % % %
#

c = int(input("Give me a number!"))
h = "% "
h_m = " %"
for i in range(0,c):
    if i % 2 == 0:
        print(c * h)
    else: 
        print (c * h_m)

File: week-07/day-01/draw-chess-table.js

// Crate a program that draws a chess table like this
//
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % % 
//  % % % %
//

var style01 = "% % % %";
var style02 = " % % % %";

var i;

for (i = 1; i < 9; i++) {
    if (i % 2 === 0) {
        console.log(style01);
    } else {
        console.log(style02);
    }
}
